maas/resource_maas_network_interface_bridge_test.go

```diff
@@ -68,7 +68,7 @@ func TestAccResourceMaasNetworkInterfaceBridge_basic(t *testing.T) {
 		resource.TestCheckResourceAttr("maas_network_interface_bridge.test", "bridge_fd", "42"),
 		resource.TestCheckResourceAttr("maas_network_interface_bridge.test", "bridge_stp", "true"),
 		resource.TestCheckResourceAttr("maas_network_interface_bridge.test", "bridge_type", "standard"),
-		resource.TestCheckResourceAttr("maas_network_interface_bridge.test", "mac_address", mac_address),
+		resource.TestCheckResourceAttr("maas_network_interface_bridge.test", "macAddress", macAddress),
```

Comment:
Same confusion with a string value

Reply:
same explanation as above.

Reply:
Hm, I thought you said that it should be "mac_address"?